cmd/exocored/root.go

```diff
@@ -132,7 +143,24 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 		a.appExport,
 		addModuleInitFlags,
 	)
-
+	startCmd, _, _ := rootCmd.Find([]string{cmdStartName})
+	startCmd.Flags().Bool(flagOracle, false, "enable oracle feeder")
+	startCmd.Flags().String(flagMnemonic, "", "set validator consensus key's mnemonic")
+	preRunE := startCmd.PreRunE
+	// add preRun to run price-feeder first before starting the node
+	startCmd.PreRunE = func(cmd *cobra.Command, args []string) error {
+		if enableOracle, _ := cmd.Flags().GetBool(flagOracle); enableOracle {
+			mnemonic, _ := cmd.Flags().GetString(flagMnemonic)
```

Comment:
Check if the **mnemonic** is empty before starting the price-feeder ?

Reply:
the priority is 
 1. cli: --mnemonic
 2. field mnemonic in config file
 3. privatekey from priv_validator_key.json
So we allow this to be empty